schema-extended/validate.py

```diff
@@ -0,0 +1,57 @@
+"""Jsonschema validation script that supports file $refs."""
+import click
+import json
+from jsonschema import RefResolver, Draft7Validator
+from jsonschema import validate, RefResolver
+from jsonschema.validators import validator_for
+
+schema_store = {}
+schema_files = [
+    "Affiliation.json",
+    "Common.json",
+    "Document.json",
+    "Funding.json",
+    "Keyword.json",
+    "LanguageCode.json",
+    "License.json",
+    "Author.json",
+    "References.json",
+    "RelatedLink.json",
+    "Role.json",
+    "Taxonomy.json",
+    "Version.json"
```

Comment:
@mhl10 did you forget to commit some of these? or are they still WIP?

Reply:
@michamos they should all be there, in the same directory as the script.

Reply:
I can't find `Funding` for example (using the `jump to` menu at the top)

Reply:
my bad, this is only the latest unmerged diff, unchanged files don't appaear here. Thanks!